hist/hist/inc/TH1.h

```diff
@@ -326,7 +326,12 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
    virtual void     GetStats(Double_t *stats) const;
    virtual Double_t GetStdDev(Int_t axis=1) const;
    virtual Double_t GetStdDevError(Int_t axis=1) const;
-   virtual Double_t GetSumOfWeights() const;
+   Double_t         GetSumOfAllWeights(const bool includeOverflow) const;
+   ////////////////////////////////////////////////////////////////////////////////
+   /// Return the sum of weights excluding under/overflows.
+   /// \note Different from TH1::GetSumOfAllWeights, that allows you to control
+   /// if you include or exclude those.
```

Comment:
This header doesn't have much documentation, so let's keep it super short to blend in.
```suggestion
   /// Return the sum of weights across all bins excluding under/overflows.
   /// \see TH1::GetSumOfAllWeights()
```